client_gui.py

```
@@ -0,0 +1,568 @@
+        self.status_var.set("Connecting...")
+        
+        def run_async_loop():
+            self.loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(self.loop)
+            
+            self.client = SecureChatClientCore(host, port)
+            self.setup_client_callbacks()
+            
+            try:
+                self.loop.run_until_complete(self.async_connect())
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)